main.go

```diff
@@ -10,33 +11,90 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var (
+	flagGitHubOrganization = flag.String("org", "default", "GitHub organization to scan.")
+	flagGitHubLabel        = flag.String("label", "default", "GitHub label marking all issues to be included.")
+)
+
 func main() {
+	flag.Parse()
+
 	token := os.Getenv("GITHUB_AUTH_TOKEN")
 	if token == "" {
 		log.Fatal("Unauthorized: No token present")
 	}
+
+	if flagGitHubOrganization == nil || *flagGitHubOrganization == "" {
+		log.Fatal("Error: Required --org flag not proided.")
+	}
+	if flagGitHubLabel == nil || *flagGitHubLabel == "" {
+		log.Fatalf("Error: Required --label flag not provided.")
+	}
+
+	log.Printf(
+		"Scanning GitHub organization %q and all issues labeled %q...",
+		*flagGitHubOrganization, *flagGitHubLabel)
+
 	ctx := context.Background()
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
 	tc := oauth2.NewClient(ctx, ts)
 	client := github.NewClient(tc)
 
 	var sumPoints int
-	fmt.Printf("\n\nISSUES - POINTS\n")
 
+	// https://developer.github.com/v3/issues/#list-issues
 	opts := &github.IssueListOptions{
+		ListOptions: github.ListOptions{
+			PerPage: 50,
+		},
 		Filter: "all",
-		Labels: []string{"20Q1-svc"},
+		State:  "open",
+		Labels: []string{*flagGitHubLabel},
 	}
-	issues, _, _ := client.Issues.ListByOrg(ctx, "pulumi", opts)
-	for _, i := range issues {
-		sizePoints := getSizeValue(i)
-		sumPoints += sizePoints
-		fmt.Printf("%s - %d\n", i.GetTitle(), sizePoints)
+
+	for {
+		issues, resp, err := client.Issues.ListByOrg(ctx, *flagGitHubOrganization, opts)
+		if err != nil {
+			log.Fatalf("error listing GitHub issues: %v", err)
+		}
+
+		// Columns
+		fmt.Printf("%-70s\t%20s\t%20s\t%8s\t%s\n",
```

Comment:
Obviously this kinda sucks. But the alternative would be to introduce more structure. Like:

```go
type columnDefinition {
   Label string
   Width int
   LeftAlign bool
}
```

But that's probably overkill. But happy to make the change if you'd prefer that over the ~unreadable value I have here. (Or if you'd like to keep the original output formatting!)